README.md

```diff
@@ -114,6 +116,14 @@ The Qt frontend includes an automatic update checker. Builds downloaded after 20
 
 ### Linux
 
+#### Flatpak
+
```

Comment:
Change this to "No support for the flatpak package will be provided, any emulator issues must be verified with the appimage or a source build before being reported. Flatpak issues can be reported at: ...."

And move it underneath the first link.